### DISCUSSION TITLE
Ajudem  no Fooocus AI,o face swap não gera rosto parecido a minha modelo

### DISCUSSION THREAD

**Discussion by Rodrigonobrex** [lllyasviel/Fooocus#3167] [General]
Eu sei que meu título tá horrível,me ajudam eu gero pelo celular mesmo. Eu boto ele pra gerar porém os rosto não fica parecido 

Eu esse pra por no face swap até botei outro modelo porém botei em outra altura etc 
![2024-06-19_10-27-00_4829](https://github.com/lllyasviel/Fooocus/assets/120036549/2fc2b379-710d-4a5b-aea4-6a2792462a67)

Olhem 

![2024-06-19_20-23-00_2296](https://github.com/lllyasviel/Fooocus/assets/120036549/9286ed19-c9f0-40d4-b091-eaaea132e5e9)
![2024-06-19_20-35-13_7004](https://github.com/lllyasviel/Fooocus/assets/120036549/c6c68394-e408-41be-bbf7-c348195be0cd)
![2024-06-19_20-47-57_7868](https://github.com/lllyasviel/Fooocus/assets/120036549/2d15f97c-6157-4b21-849c-9767280b7861)
![2024-06-19_20-51-46_5185](https://github.com/lllyasviel/Fooocus/assets/120036549/cfe6d496-611e-4fef-9214-0b382be114bc)

Alguém der uma ajuda,eu criei ele como influencer e tenho uma que eu criei de mulher,ele criei e botei 18 anos e tenho outro influencer também porém a mesma coisa,o rosto não fica parecido

**Comment by PeterJared** [comment]
Even Fooocus does a decent job with face swap, you won't get a very accurate one. Also it depends on the faces. Sometimes gets a character likeness better than others. 
If you want perfect likeness you can yet generate your models in Fooocus, and even use face swap as a starting point and then send to an app that is more specialized. 
There is a recent post asking about what is the best app for this and even no one answered yet, I guess it gives a good idea what are the best apps for it. 
Personally I only used remaker.ai, that is superior to Fooocus for face swapping and quite simple to use , even I'm nearly sure there are better apps.

**Reply by Rodrigonobrex** [reply]
Obrigado pela dica,eu gosto muito do remaker,vou fazer isso.
